doc/classes/OpenXRExtensionWrapperExtension.xml

```diff
@@ -0,0 +1,175 @@
+			</description>
+		</method>
+		<method name="_on_process" qualifiers="virtual">
+			<return type="void" />
+			<description>
+				Called as part of the OpenXR process handling.
+				This happens right before physics process and normal processing is run.
+				This is when controller data is queried and made available to game logic.
+			</description>
+		</method>
+		<method name="_on_register_metadata" qualifiers="virtual">
+			<return type="void" />
+			<description>
+				Allows extensions to register additional controller metadata.
+				This function is called even when [code]OpenXRAPI[/code] is not constructured as the metadata needs to be available to the editor.
+				Extensions should also provide metadata regardless of whether they are supported on the host system as the controller data is used to setup action maps for users who may have access to the relevant hardware.
+			</description>
+		</method>
+		<method name="_on_session_created" qualifiers="virtual">
+			<return type="void" />
+			<param index="0" name="session" type="int" />
+			<description>
+				Called right after we've successfully created the OpenXR session.
+			</description>
+		</method>
+		<method name="_on_session_destroyed" qualifiers="virtual">
+			<return type="void" />
+			<description>
+				Called right before the OpenXR session is destroyed.
+			</description>
+		</method>
+		<method name="_on_state_exiting" qualifiers="virtual">
+			<return type="void" />
+			<description>
+				Called when the OpenXR session state is changed to exiting.
+			</description>
+		</method>
+		<method name="_on_state_focused" qualifiers="virtual">
+			<return type="void" />
+			<description>
+				Called when the OpenXR session state is changed to focused.
+				This state is the active state when the game runs.
+			</description>
+		</method>
+		<method name="_on_state_idle" qualifiers="virtual">
+			<return type="void" />
+			<description>
+				Called when the OpenXR session state is changed to idle.
+			</description>
+		</method>
+		<method name="_on_state_loss_pending" qualifiers="virtual">
+			<return type="void" />
+			<description>
+				Called when the OpenXR session state is changed to loss pending.
+			</description>
+		</method>
+		<method name="_on_state_ready" qualifiers="virtual">
+			<return type="void" />
+			<description>
+				Called when the OpenXR session state is changed to ready.
+				This means OpenXR is ready to set up the session.
+			</description>
+		</method>
+		<method name="_on_state_stopping" qualifiers="virtual">
+			<return type="void" />
+			<description>
+				Called when the OpenXR session state is changed to stopping.
+			</description>
+		</method>
+		<method name="_on_state_synchronized" qualifiers="virtual">
+			<return type="void" />
+			<description>
+				Called when the OpenXR session state is changed to synchronized.
+				OpenXR also returns to this state when the application looses focus.
```

Comment:
```suggestion
				Called when the OpenXR session state is changed to synchronized. OpenXR also returns to this state when the application loses focus.
```

You need to avoid new lines in the docs, unless you want to create a new paragraph (because that's what a new line here does). In this case it should be the same paragraph, as the sentences are short and the context is the same.

Also a typo.